src/axom/sidre/core/DataStore.hpp

```diff
@@ -88,6 +88,21 @@ class DataStore
    */
   const Group* getRoot() const { return m_RootGroup; };
 
+  /// Return whether a Conduit error occurred.
+  bool getConduitErrorOccurred() const { return m_conduit_error_occurred; };
+  void setConduitErrorOccurred(bool flag) const
```

Comment:
Should this be a public method? If used outside of DataStore class methods, then maybe a fried of the methods in which it is called?

Reply:
I made the get/set ConduitErrorOccurred and ConduitErrors methods public, and I think they should be.  It gives codes information about how the DataStore is working with the operating system and other libraries.

Reply:
I understand the usefulness of enabling users to query the error state. It's not clear to me whether it's a good idea to allow users to change the error state. Thoughts?